app/views/templates/script/package_action.erb

```diff
@@ -11,6 +11,12 @@ template_inputs:
   input_type: user
   required: false
   advanced: true
+- name: refresh_subscription
+  description: Refresh subscription-manager state. Default is "false".
+  input_type: user
+  required: false
+  advanced: true
+  options: "false\r\ntrue"
```

Comment:
Now it defaults to false, is that intentional?

Reply:
Yes. The default for all OS was false previously. Only for Debian/Ubuntu it was true. Therefore, I think its better that this feature is off for all OS and need to be turned on if needed.